service/src/test/java/com/epam/esm/service/impl/TagServiceImplTest.java

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
@gkttk Why do we need this test class if all code is commented?

Reply:
commit da349eb